duckduckgo/Android
# Enable GPC by Default

Task/Issue URL: https://app.asana.com/0/1197921851987065/1199132975397325/f
Tech Design URL: 
CC: https://app.asana.com/0/0/1199349637642694/f

**Description**:
This PR enables GPC by default to new users.

**Steps to test this PR**:

**Setting enabled**
1. Fresh install
1. Go to settings
1. Ensure GPC is enabled

**Signal present and back navigation**
1. Go to globalprivacycontrol.org
1. At the top of the page it should say `GPC signal detected`.
1. Click on `test against the reference server`
1. glitch.me site should load and both the header and DOM signals should show as present.
1. Inspect the tab by going to chrome://inspect in your desktop browser, select the tab opened in your phone and click network.
1. Back in the app, go to the bottom of the site and click to `View page source` 
1. In the inspected tab, in networks, select the first request and inspect the headers. The sec-gpc header should be set to `1`.
1. In the phone tab, click the back button.
1. The header and DOM should still show as present.
1. Click the back button and the sign should still be detected
1. Click back and you should be taken to the home screen.
1. Click forward until you reach github again navigating as expected and with the header set correctly.

**Blank link**
1. Go to https://www.w3schools.com/code/tryit.asp?filename=GL0N23JVASNE and execute the code.
1. App should not crash

**Blank link signal present**
1. Go to https://www.w3schools.com/code/tryit.asp?filename=GLNSDACQ1Y8L and execute the code.
1. You should be taken to the glitch site and the header and DOM signals should be present.

**Page rewrite**
1. Open DDG app (and press fire to be sure we clear the cache)
1. Visit https://duckduckgo.github.io/privacy-test-pages/security/spoof-js-page-rewrite.html
1. Press Start
1. Ensure url is updated to duck.co and fake content is visible for 1 second
1. Ensure web content hides after 1 second

**Fireproof**
1. Enable the fireproof toggle in Settings -> Fireproof Websites
1. Go to different sites, i.e. facebook, twitter, netflix, wsj.com
1. After login in those sites you should see the fireproof dialog

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
